### PR TITLE
eliminated add-script use

### PR DESCRIPTION
**Pull request by pflee1989**
Co-authored-by: Chris <chris-chilton@hotmail.com>

# Pull Request Template

## Description

Eliminated a function that relied on an unused class method called add-script. This is a small change toward cleaner code. 

Fixes # (issue)
N/A

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test: We tested the code locally to see if the functions involved worked, and the Twitterbot still works. 

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

## The Link to Video
https://drive.google.com/file/d/1eiZdOmPkwbAf-5X7CWTit_e9S1-tk2qJ/view?usp=sharing

### CODE REVIEW

**Review by mikecarrier4 — APPROVED** [review]
Good work on simplifying the code base.  The code looks concise and well formatted

**Review by kcjerrell — APPROVED** [review]

**Review by paulstgermain — CHANGES_REQUESTED** [review]
This looks like a solid, needed PR, however I'd love for you to record the standard Loom video walking through the code and the changes made. These don't need to be overly long unless necessary - just the what, why, and how of this PR would be great! 

(PS - you can include your co-contributor on this video walkthrough if it helps. :) )

**Comment by pflee1989** [comment]
https://drive.google.com/file/d/1eiZdOmPkwbAf-5X7CWTit_e9S1-tk2qJ/view?usp=sharing

**Review by paulstgermain — APPROVED** [review]
## Video Walkthrough Feedback

This video walkthrough for this PR could have used a _lot_ more detail - these serve 2 purposes:

1. Verifying the changes made to the codebase
2. Giving you all chance to get experience talking through your work ahead of interviews

That said, coming at this as someone with a critical perspective like an interviewer, or senior dev/codebase manager, this explanation didn't describe or show the work done at all aside from leaning on a senior teammate to describe the removal of unused code. This is perfectly acceptable, but I would have liked to see you walk through some of the other changes made, such as the change made in the `bot.py` file, line 55.

## Code Review

The changes to the code itself are solid, but again, especially as someone with admittedly less Python/DS experience, I would have liked more of an explanation for some of these changes.


All of that said, with the knowledge that this was worked on with the help of a senior teammate I'm going to go ahead an approve this PR as a needed change before shipping this week, but I'd love to see you implement some of this feedback moving forward!

To recap: Show your work as you try and explain it - this helps you keep on track and not skimp over any material that should be discussed.
